source/extensions/filters/network/redis_proxy/conn_pool_impl.cc

```diff
@@ -95,6 +112,9 @@ void InstanceImpl::ThreadLocalPool::onHostsRemoved(
       // we just close the connection. This will fail any pending requests.
       it->second->redis_client_->close();
     }
+    if (host->address()) {
```

Comment:
Can this ever be nullptr? I don't think so?

Reply:
looking at the code as a whole, it appears that it would be possible to create a malformed Host, but it looks like it would be a bug. I'll put in protection in any case.

Reply:
Right, no need to check this.

Reply:
ok